examples/hello/hello.cpp

```diff
@@ -85,28 +85,28 @@ ShaderProgram* loadShaderProgram(Renderer* renderer)
     // translation unit in which that function can be found, and the stage
     // that we need to compile for (e.g., vertex, fragment, geometry, ...).
     //
-    char const* vertexEntryPointName    = "vertexMain";
-    char const* fragmentEntryPointName  = "fragmentMain";
-    int vertexIndex   = spAddEntryPoint(slangRequest, translationUnitIndex, vertexEntryPointName,   SLANG_STAGE_VERTEX);
+    char const* vertexEntryPointName = "vertexMain";
```

Comment:
Note: this and a few other places seem to have been either manually or automatically formatted, ans as a result there's a bunch more diffs here than there need to be. I wouldn't tend to point it out, but in this case they are changes *away* from the style that I tend to prefer (vertically align when possible, don't put space between control-flow statement keyword and opening parens, etc.).

Just nit-picking here, since I don't plan to ask for this to be reverted.